content/scout/guides/vex.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Suppress image vulnerabilities with VEX
+description: An introduction to using VEX with Docker Scout
+keywords: scout, vex, Vulnerability Exploitability eXchange, openvex, vulnerabilities, cves
+---
+
+In this guide, you will learn about:
+
+- How VEX can help you suppress non-applicable or fixed vulnerabilities found in your images
+- How to create VEX documents and statements
+- How to apply and consume VEX data with Docker Scout
+
+## Prerequisites
```

Comment:
Can users try this with Docker Personal Or, do they need any of the paid subscriptions?

Reply:
No subscription required. Docker Scout doesn't gate features behind subscriptions, only repository count.